modules/unauthenticated-users-cluster-role-binding.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * authentication/impersonating-system-admin.adoc
+// * cicd/builds/triggering-builds-build-hooks.adoc
+// * authentication/tokens-scoping.adoc
+// * authentication/managing-oauth-access-tokens.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-cluster-role-bindings_{context}"]
+= Adding unauthenticated groups to cluster role bindings
+
+As a cluster administrator, you can add unauthenticated users to the following cluster role bindings in {product-title}. Unauthenticated users do not have access to most cluster role bindings by default.
+
+* `system:scope-impersonation`
+* `system:webhook`
+* `system:oauth-token-deleter`
+* `self-access-reviewer`
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying unauthenticated access.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+[Note]
+====
+Replace the `<cluster_role_binding>` variable with the cluster role binding you want to update.
+====
+
+. Create a YAML file named `add-<cluster_role_binding>-unauth.yaml` and add the following content:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+ annotations:
+   rbac.authorization.kubernetes.io/autoupdate: "true"
+ name: restore-<cluster_role_binding>-unauth
+roleRef:
+ apiGroup: rbac.authorization.k8s.io
+ kind: ClusterRole
+ name: <cluster_role_binding>
```

Comment:
This is the name of the Cluster Role, not the binding. So it would need to be 

```yaml
  name: "system:webhook"
```.

It is important to put `system:webhook` in `"`, otherwise it is misinterpreted.

Reply:
```mermaid
graph LR
    A[system:unauthenticated]
    C[system:webhook]

    A -- restore-webhook-unauth --> C
```